test/cases/expressions/xhp-pound-inline.hack

```diff
@@ -0,0 +1 @@
+return <body> # </body>;
```

Comment:
nit: I know we did this in a few other tests, but we don't really need the `return` here. Unless we're adding other statements/expressions to specifically to test their behavior when mixed with other expressions, we should leave them out.

It's a small difference, but I think it adds up for how fast you can read `.exp` output at a glance.

I'd also add in a few more tests (can go in the same file) to make sure we don't regress on them: 1. a test with `#` on it's own line, 2) a test with `//` (we didn't run into this specific problem, but the result should be the same as `#`, 3) a test text following `#` (e.g. `#hello`), 4) a test with no spaces between tags and the comment (e.g. `#hello</body>) (I think this can have an affect on whether something is parsed as a comment or not)

Reply:
Haha the `return` nit was my fault for starting as a trend on that other PR

But also #trendsetter 🏆

Reply:
Yeah no biggie 😄  Saw that in your PR and wasn't sure whether to comment, how to word it, and whether it was being too nitpicky. Just took me a few days to put the words/thoughts together 😁

Reply:
Should probably add a style guide to the repo 🤔